minerl/herobraine/wrappers/video_recording_wrapper.py

```diff
@@ -0,0 +1,81 @@
+import os
+import pathlib
+
+import numpy as np
+from gym import Wrapper
+import cv2
+
+
+class _VideoWriter:
+    """Helper class for writing MineRL images into mp4 video via `cv2`."""
+    fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+
+    def __init__(self, video_width, video_height, fps=20.0):
+        self.video_width = video_width
+        self.video_height = video_height
+        self.fps = fps
+        self.out = None
+
+    def is_open(self):
+        return self.out is not None
+
+    def open(self, path):
+        if self.is_open():
+            raise ValueError("Call close() first.")
+        self.out = cv2.VideoWriter(
+            str(path),
+            self.fourcc,
+            self.fps,
+            (self.video_height, self.video_width),
+        )
+
+    def write_rgb_image(self, img: np.ndarray):
+        assert img.ndim == 3
+        self.out.write(np.flip(img, axis=-1))
+
+    def close(self):
+        if self.out is not None:
+            self.out.release()
```

Comment:
Just checking - is release the correct close action here?

Reply:
Yeah, `cv2` is a C++ bindings package, and the naming convention there is `release()`